docs/components/attention/react.md

```diff
@@ -0,0 +1,111 @@
+## Import
+
+```js
+import { Attention } from '@warp-ds/react';
+```
+
+## Visual options
+
+### Callout
+
+```js
+<div className="flex items-center">
+  <Box info>
+    <h1>I am a box full of info</h1>
+  </Box>
+  <Attention callout placement="right" isShowing={true}>
+    <p>I'm a callout because that box over there is new or something</p>
+  </Attention>
+</div>
+```
+
+### Tooltip
+
+```js
+function Example() {
+  const [show, setShow] = React.useState(false);
+  const targetEl = React.useRef();
+
+  return (
+    <div>
+      <h1
+        className="w-max mb-0"
+        ref={targetEl}
+        onMouseEnter={() => setShow(true)}
+        onMouseLeave={() => setShow(false)}
+      >
+        hover this for useless info
+      </h1>
+      <Attention
+        tooltip
+        placement="bottom"
+        targetEl={targetEl}
+        isShowing={show}
+      >
+        <p>lol i am a popover</p>
+      </Attention>
+    </div>
+  );
+}
+```
+
+### Popover
+
+```js
+function Example() {
+  const [show, setShow] = React.useState(false);
+  const containerRef = React.useRef();
+  const targetEl = React.useRef();
+
+  React.useEffect(() => {
+    function onBlurHandler(e) {
+      if (containerRef.current && !containerRef.current.contains(e.target)) {
+        setShow(false);
+      }
+    }
+    document.addEventListener('mousedown', onBlurHandler);
+    return () => {
+      document.removeEventListener('mousedown', onBlurHandler);
+    };
+  });
+
+  return (
+    <div ref={containerRef}>
+      <Button
+        small
+        utility
+        onClick={() => setShow(!show)}
+        className="w-max mb-0"
+        ref={targetEl}
+      >
+        Open menu
+      </Button>
+      <Attention
+        popover
+        placement="bottom"
+        targetEl={targetEl}
+        isShowing={show}
+      >
+        <ul className="bg-white w-full text-center">
+          <li
+            tabIndex={0}
+            className="p-2 px-40 hover:bg-blue-200 hover:text-blue-500 cursor-pointer"
```

Comment:
I'm a bit worried that we display utility classes here that no longer are valid in Warp. I don't think it's a problem that we remove any such classes from tech-docs, since we don't show the actual components anyway. What do you think?